src/frontend/src/components/public/PIAFormTabs/review/index.tsx

```diff
@@ -111,7 +110,20 @@ const PIAReview = ({ printPreview }: IReviewProps) => {
       true,
     );
   };
+  const allowUserReviewProgramArea = () => {
+    const userGuid = getGUID();
+    const selectedRoles = reviewForm?.programArea?.selectedRoles;
```

Comment:
please add a null check after this. If selectedRoles object does not exist return true.

Reply:
added the check